doc/api/buffer.markdown

```diff
@@ -130,10 +130,7 @@ the list together.
 If the list has no items, or if the totalLength is 0, then it returns a
 zero-length buffer.
 
-If the list has exactly one item, then the first item of the list is
-returned.
-
-If the list has more than one item, then a new Buffer is created.
+If the list has one or more items, then a new Buffer is created.
```

Comment:
Sounds good to me. Seems it existed in the first place to explain the exceptional behavior.